lib/jekyll/renderer.rb

```diff
@@ -131,7 +131,9 @@ def invalid_layout?(layout)
     # Returns the content placed in the Liquid-rendered layouts
     def place_in_layouts(content, payload, info)
       output = content.dup
-      layout = site.layouts[document.data["layout"]]
+      # Load first layout from list, for now
+      layouts = site.layouts[document.data["layout"]]
+      layout = layouts ? layouts.first : nil
```

Comment:
I'd love use the Ruby 2.3 `&.` operator, but we can't do that yet I guess because of support for older rubies. Need to check if activesupport is available for `#try`, but I've not done that yet.